pkg/util/netlink.go

```diff
@@ -0,0 +1,135 @@
+package util
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/vishvananda/netlink"
+)
+
+func ModeFromString(s string) (netlink.MacvlanMode, error) {
+	switch s {
+	case "", "bridge":
+		return netlink.MACVLAN_MODE_BRIDGE, nil
+	case "private":
+		return netlink.MACVLAN_MODE_PRIVATE, nil
+	case "vepa":
+		return netlink.MACVLAN_MODE_VEPA, nil
+	default:
+		return 0, fmt.Errorf("unknown macvtap mode: %q", s)
+	}
+}
+
+func CreateMacvtap(name string, master string, mode string) (int, error) {
+	// attempt to delete any previously existing link
```

Comment:
To make it clear what is the `int` you are returning representing, would you please initialize the variable here with a proper name and default value `0`?